File: library/src/main/java/com/smartnsoft/smartapprating/SmartAppRatingActivity.java
package com.smartnsoft.smartapprating;

import android.os.Bundle;

/**
 *
 * @author Adrien Vitti
 * @since 2018.02.13
 */

public final class SmartAppRatingActivity
    extends AbstractSmartAppRatingActivity
{

  @Override
  public int getVersionCode()
  {
    return 0;
  }

  @Override
  public String getDateForAnalytics()
  {
    return null;
  }

  @Override
  public void sendAnalyticsEvent(String eventName, Bundle bundle)
  {

  }

}

File: core/src/main/java/com/smartnsoft/smartapprating/SmartAppRatingActivity.kt
package com.smartnsoft.smartapprating

import android.os.Bundle

/**
 *
 * @author Adrien Vitti
 * @since 2018.02.13
 */

class SmartAppRatingActivity : AbstractSmartAppRatingActivity()
{

  override fun getVersionCode(): Int
  {
    return 0
  }

  override fun getDateForAnalytics(): String?
  {
    return null
  }

  override fun sendAnalyticsEvent(eventName: String, bundle: Bundle)
  {

  }

}
